native/packages/react-native-bpk-component-button/src/BpkButton-styles.js

```diff
@@ -0,0 +1,194 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Platform, StyleSheet } from 'react-native';
+
+const IOS_TOKENS = require('bpk-tokens/tokens/ios/base.react.native.common.js');
+const ANDROID_TOKENS = require('bpk-tokens/tokens/android/base.react.native.common.js');
```

Comment:
Matt and I thought it was best to in-line these so that it doesn't actually import everything for both platforms. We're not even sure if this optimises everything but it might?!